site/components/Showcases.tsx

```diff
@@ -22,11 +22,20 @@ const items = [
     sourceUrl: 'https://github.com/FCSCOpendata/frontend',
   },
   {
-    title: 'Datahub Open Data',
-    href: 'https://opendata.datahub.io/',
-    image: '/images/showcases/datahub.webp',
-    description: 'Demo Data Portal by DataHub',
+    title: 'Frictionless Data',
+    href: 'https://frictionlessdata.io',
```

Comment:
@lucasmbispo it should link to showcase page, I would recommend linking to one popular dataset showcase on datahub.io, for example you can use https://datahub.io/core/co2-ppm (it has some charts)